toolsrc/src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -312,30 +291,49 @@ namespace vcpkg
                 {JSON_SWITCH, &VcpkgCmdArguments::json},
             };
 
+            Optional<StringView> lookahead;
+            {
+                auto next = it;
+                ++next;
+                if (next != arg_last)
+                {
+                    lookahead = *next;
+                }
+            }
```

Comment:
This is a pointer, and therefore this can be much simpler:

```suggestion
            Optional<StringView> lookahead = (it + 1 != arg_last) ? *(it + 1) : Optional<StringView>(nullopt);
```

Reply:
I hate ternaries so I didn't do exactly that but I did avoid the `next` variable.